cfme/tests/services/test_different_dialogs_in_catalogs.py

```diff
@@ -477,7 +551,33 @@ def test_dialog_elements_should_display_default_value(element_type):
             1.
             2.
             3. Default values should be shown
-    Bugzilla:
-        1385898
     """
-    pass
+    sd, element_data = service_dialog
+    service_catalogs = ServiceCatalogs(
+        appliance, generic_catalog_item.catalog, generic_catalog_item.name
+    )
+    view = navigate_to(service_catalogs, "Order")
+    ele_name = element_data["element_information"]["ele_name"]
+    wt, value = WIDGETS[widget_name]
+
+    if widget_name in ["Text Box", "Text Area"]:
```

Comment:
This is also same for Dropdown and ["Timepicker", "Datepicker"]

     if widget_name in ["Text Box", "Text Area", "Dropdown", "Timepicker", "Datepicker"]:
        assert getattr(view.fields(ele_name), wt).read() == value